utils/fs.go

```diff
@@ -8,11 +8,9 @@ import (
 
 // FileExists checks if the file exists, gracefully handling ENOENT.
 func FileExists(path string) bool {
-	if _, err := os.Stat(path); err != nil {
-		if os.IsNotExist(err) {
-			return false
-		}
-		glog.Fatalf("Failed to stat %s: %v", path, err)
```

Comment:
> but FileExists shouldn't return true if there are errors

It doesn't, this line would abort the process right?

Reply:
yeah, it exits 255 indeed, I'm back to "reboot-needed file was there but it really shouldn't have been"

Reply:
Agreed. `Fatalf` would exit.